README.md

```diff
@@ -681,3 +437,120 @@ thereafter it should be possible to use one of the different checkpatch targets
 in the [Makefile](Makefile). There are targets for checking all files, checking
 against latest commit, against a certain base-commit etc. For the details, read
 the [Makefile](Makefile).
+
+## 6. repo manifests
```

Comment:
Right @pascal-brand-st-dev , that is what I also was thinking (the first option). When providing the link at the manifest/README.md, we can also just highlight that GP attendees, should follow that particular link. I can make the changes there when this particular patch has been merged.